src/native/Native.cpp

```diff
@@ -106,6 +111,111 @@ void _CheckInterrupts() {
 
 
 
+//Copied over from OOT - needs cleanup
+struct Gfx;
+struct OSThread;
+typedef void* OSMesg;
+
+typedef struct OSMesgQueue {
+    /* 0x00 */ OSThread* mtqueue;
+    /* 0x04 */ OSThread* fullqueue;
+    /* 0x08 */ s32 validCount;
```

Comment:
Let’s remove this, and just include the original files directly.  Add the ooot include path to the project.